code/core/src/core-server/utils/get-mocked-props-for-args.ts

```
@@ -0,0 +1,282 @@
+      return [generateMockValueFromSBType(sbType.value, propName)];
+    }
+
+    case 'tuple':
+      return sbType.value.map((el) => generateMockValueFromSBType(el));
+
+    case 'enum':
+      return sbType.value[0] ?? propName;
+
+    case 'intersection':
+      // For intersections, combine all object types
+      const objectTypes = sbType.value.filter((t) => t.name === 'object');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)